src/main/resources/schemas/img-metadata.schema.json

```
@@ -0,0 +1,205 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "http://qbic.life/v1/img-metadata.schema.json",
+  "title": "Imaging Metadata Model",
+  "description": "Describes the imaging metadata model.",
+  "type": "object",
+  "properties": {
+    "image_filename": {
+      "type": "string",
+      "minLength": 1
+    },
+    "imaging_modality": {
+      "type": "string",
+      "minLength": 1
+    },
+    "imaging_date": {
+      "type": "string",
+      "pattern": "^[0-9]{2}.[0-9]{2}.[0-9]{4}$"
```

Awesome! I suggest to make it a bit stricter:
```suggestion
      "pattern": "^[0-3][0-9].[0-2][0-9].[1-9][0-9]{3}$"
```